.github/workflows/ompi_nvidia.yaml

```diff
@@ -16,19 +18,21 @@ jobs:
     - name: Deployment infrastructure
       run: /start deploy
   build:
+    if: github.repository == 'open-mpi/ompi'
     needs: [deployment]
     runs-on: [self-hosted, linux, x64, nvidia]
     steps:
     - name: Building OMPI,UCX and tests
       run: /start build
   test:
+    if: github.repository == 'open-mpi/ompi'
     needs: [deployment, build]
     runs-on: [self-hosted, linux, x64, nvidia]
     steps:
     - name: Running tests
       run: /start test
   clean:
-    if: ${{ always() }}
+    if: (github.repository == 'open-mpi/ompi') && always()
```

Comment:
I think you can get rid of the `&& always()`, right?

Reply:
Maybe, but I wasn't sure

Reply:
> Maybe, but I wasn't sure

I am not an expert on Github actions, but a function named `always()` seems to imply that it is always true, and when you've now got another condition in the expression, it feels like you don't need the `always()` (i.e., you only need `always()` if you have an expression with just a single condition).  You might want to test this to be sure.

Reply:
There are examples of different behaviors with _always(_) and without:
[**with always():**](https://github.com/open-mpi/ompi/actions/runs/6088240151)
[**without always()**](https://github.com/open-mpi/ompi/actions/runs/6088181437)
_always()_ runs even job is canceled, but the condition is false w/o _always()_

Reply:
Ok.  If the `always()` is needed, then it is worth putting that in a comment, because that's not intuitive -- and we wouldn't want someone to remove the `always()` at some point in the future because they think it's not relevant.

Reply:
Done